test/E2E_Test/E2ETestBase.cs

```diff
@@ -12,24 +13,26 @@ namespace Microsoft.VisualStudio.Web.CodeGeneration.E2E_Test
 {
     public class E2ETestBase
```

Comment:
Lot of extra changes in this PR. Is this the bulk of what was changed to enable E2E tests?